mixer/cmd/root.go

```diff
@@ -105,6 +108,18 @@ var RootCmd = &cobra.Command{
 		// If so: inform, stage, and exit.
 		// If not: run command in container and cancel pre-run
 		if !cmdContains(cmd, "format-bump") && !cmdContains(cmd, "upstream-format") && cmdContains(cmd, "build") {
+			// --offline=true AND --native=false, try to see if container exists
+			if !networkCheck && !builder.Native {
```

Comment:
Why not just look at `builder.Offline` here, rather than `networkCheck`? You say this is based on `--offline=true`, but you're routing that through `networkCheck`, which could be set for other reasons. Feels like a bad coupling to me.